src/network/mcpe/protocol/types/event/FishBucketedEventData.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+*/
+
+declare(strict_types=1);
+
+namespace pocketmine\network\mcpe\protocol\types\event;
+
+use pocketmine\network\mcpe\protocol\EventPacket;
+use pocketmine\network\mcpe\serializer\NetworkBinaryStream;
+
+final class FishBucketedEventData implements EventData{
+	/** @var int */
+	public $pattern;
+	/** @var int */
+	public $preset;
+	/** @var int */
+	public $type;
+	/** @var bool */
+	public $released;
```

Comment:
All of these fields have ambiguous names. It is not clear at a glance what the purpose of any of them are.